vitrina/orgs/views.py

```diff
@@ -54,3 +65,58 @@ def get_context_data(self, **kwargs):
         organization: Organization = self.object
         context_data['members'] = organization.representative_set.all()
         return context_data
+
+
+class RepresentativeCreateView(LoginRequiredMixin, PermissionRequiredMixin, CreateView):
+    model = Representative
+    form_class = RepresentativeCreateForm
+    template_name = 'base_form.html'
+
+    def has_permission(self):
+        return self.request.user.role == Representative.COORDINATOR
```

Comment:
User must be not just any coordinator, but coordinator of this organization. Also, `user.is_staff` should also be able to add representative of any organization.

And move this into `services.py`, because permission checking will be used in multiple places.